pkg/volume/iscsi/iscsi_util.go

```diff
@@ -205,6 +205,16 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) error {
 	iscsiTransport = extractTransportname(string(out))
 
 	bkpPortal := b.Portals
+
+	// create new iface and copy parameters pre-configured iface to the created iface
+	if b.InitiatorName != "" {
+		err = cloneIface(b, bkpPortal[0])
+		if err == nil {
+			// update iface name with <target portal>:<volume name>
+			b.Iface = bkpPortal[0] + ":" + b.volName
```

Comment:
the iface is later used to construct the mount point. Adding a long volume name could makes the path go beyond max path or file length.